pkg/products/threescale/reconciler.go

```diff
@@ -922,6 +922,9 @@ func (r *Reconciler) reconcileExternalDatasources(ctx context.Context, serverCli
 	if err != nil {
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to reconcile postgres request: %w", err)
 	}
+	if postgres != nil {
+		postgres.Spec.ApplyImmediately = true
+	}
```

Comment:
@austincunningham this update to the postgres happens after the ReconcilePostgres call so I think this wont get reflected to the cluster.
It's possible that you can put it inside the modify function above. e.g. before line 920 so that it gets modified inside. Although I think a better way to do that is to pass it into that function and have it update as it's a standard param of the spec. For now though I think passing it via the modifyfunc should be sufficient.